.github/workflows/tests.yaml

```diff
@@ -0,0 +1,55 @@
+name: Tests
+
+on:
+    push:
+    pull_request:
+        branches: [ master ]
+    workflow_dispatch:
+
+jobs:
+    phpunit:
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                php: [ '8.1', '8.2' ]
+                symfony: [ '5.4', '6.2' ]
+                dependencies: [ 'highest', 'lowest' ]
+        env:
+            SYMFONY_REQUIRE: ${{ matrix.symfony }}.*
+        name: PHP ${{ matrix.php }} & Symfony ${{ matrix.symfony }}${{ matrix.dependencies == 'lowest' && ' (lowest)' || '' }} Test
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v3
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    ini-values: zend.exception_ignore_args=false
+                    tools: flex
+
+            -   name: Validate "composer.json" schema
```

Comment:
@HypeMC `ramsey/composer-install` already does this step for you:

* https://github.com/ramsey/composer-install/blob/e618aa969d3a7b4f5b78f96968352d91c4c487ce/bin/composer_paths.sh#L38

Reply:
Good, catch, fixed